doc/userguide_datetimes.rst

```diff
@@ -0,0 +1,48 @@
+.. _userguide_datetimes
```

Comment:
Same comment about the label. I fear that it is not placed right above the section title and thus does not actually create a label (also, I think that it should have a colon at the end)